src/main/scala/org/renci/chemotext/Main.scala

```diff
@@ -29,6 +29,8 @@ import akka.stream.scaladsl._
 import io.scigraph.annotation.EntityAnnotation
 import io.scigraph.annotation.EntityFormatConfiguration
 
+case class ArticleInfo(pmid:String,info:String,meshTermIDs:Set[String])
```

Comment:
@ErikOverdahl I would have a separate parameter for the year, since we want to output a specific triple for this.